app/Jobs/CloneImageThumbnails.php

```diff
@@ -24,18 +24,47 @@ class CloneImageThumbnails extends Job implements ShouldQueue
      */
     public $copyPrefix;
 
-    public function __construct(String $prefix, String $copyPrefix)
+    /**
+     * Cloned image of cloned volume
+     * @var Image
+     */
+    public $image;
+
+    public function __construct(Image $img, String $prefix)
     {
+        $this->image = $img;
         $this->prefix = $prefix;
-        $this->copyPrefix = $copyPrefix;
+        $this->copyPrefix = fragment_uuid_path($img->uuid);
     }
 
     public function handle()
+    {
+        $diskThumb = Storage::disk(config('thumbnails.storage_disk'));
+        $diskTiles = Storage::disk(config('image.tiles.disk'));
+
+        if (!$this->hasThumbnail($diskThumb) || !$this->hasTiledImages($diskTiles)) {
+            ProcessNewImage::dispatch($this->image);
+            return;
+        }
```

Comment:
Not all images require tiling. Have a look at `ProcessNewImage::shouldBeTiled()`.